app/controllers/users_controller.rb

```diff
@@ -0,0 +1,38 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```

Comment:
- [ ]  It is always a good idea to remove unused code, This will make your code much clear and more readable. So please remove all commented out code.